### PR TITLE
add release v1.10.0 notes

### PR DESCRIPTION
**Pull request by denis-tingaikin**

### CODE REVIEW

File: content/docs/releases/v1.10.0.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+Apr  3 13:24:25.406 [INFO] [cmd:vpp] libbpf: Kernel error message: veth: Peer MTU is too large to set XDP
+Apr  3 13:24:25.406 [INFO] [cmd:vpp] vpp[10508]: af_xdp: af_xdp_load_program: bpf_set_link_xdp_fd(eth0) failed: Numerical result out of range
+Apr  3 13:24:26.563 [ERRO] [cmd:/bin/forwarder] [duration:18.015838ms] [hostIfName:eth0] [vppapi:AfXdpCreate] VPPApiError: System call error #6 (-16)
+panic: error: VPPApiError: System call error #6 (-16)
+```
+
+[See more details](https://github.com/networkservicemesh/integration-k8s-aws/issues/356)
+
+
+### Kubernetes installation
```
Comment (glazychev-art) [inline content/docs/releases/v1.10.0.md:168]: Might be worth clarifying: "Equinix Metal kubernetes installation"

Comment (denis-tingaikin) [reply content/docs/releases/v1.10.0.md]: Fixed

File: content/docs/releases/v1.10.0.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+### Replace govpp from nsm vpp appss to nsm/govpp
+
+In NSM it's super important to use latest patches or be able to quickly fix some issues in vpp. By this reason we need to start use govpp from nsm organization that improves control of the vpp components in the project.
+
+[See more details](https://github.com/networkservicemesh/govpp/issues/2)
+
+
+### Add a new example when one nsc connects to two nses with different services
+
+Status: done/resolved/fixed.
```
[the remark this answers is not in the document]
Comment (denis-tingaikin) [reply content/docs/releases/v1.10.0.md]: Fixed

File: content/docs/releases/v1.10.0.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+
+### cmd-admission-webhook-k8s generates client IDs properly only for Kind: Pod
+
+cmd-admission-webhook-k8s generated client IDs properly **only if client kind is Pod**.
+
+[See more details](https://github.com/networkservicemesh/deployments-k8s/issues/9170)
+
+
+### AWS: AF_XDP support
```
[the remark this answers is not in the document]
Comment (denis-tingaikin) [reply content/docs/releases/v1.10.0.md]: Fixed

**Review by glazychev-art — APPROVED** [review]

**Pull request merged**